pkg/config/config.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"gopkg.in/yaml.v2"
+)
+
+type Worker struct {
+	Count int `yaml:"count"`
+}
+
+type General struct {
+	Development bool `yaml:"development"`
+}
+type Config struct {
+	DB      db.DB   `yaml:"db"`
+	Worker  Worker  `yaml:"worker"`
+	General General `yaml:"general"`
+}
```

Comment:
As we are talking about config a better approach might be to have a more compound type definition.
Also, please define all parts of the config in this package (db.Db) 

```suggestion
type Worker 

type General 
type Config struct {
	DB      struct {
	    .....
	}   `yaml:"db"`
	Worker  struct {
	        Count int `yaml:"count"`
        }  `yaml:"worker"`
	General struct {
	        Development bool `yaml:"development"`
        }`yaml:"general"`
}
```

Reply:
Implement this but created a different struct for accessing it and for easy maintaince.